change/@fluentui-react-charting-a75da9fd-a90e-4ca9-818e-c51075622f51.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33518&commitId=7349e44ed916170dbfee10876757c52d07a48b1b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Action (compat)</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action (compat).Checked.chromium.png|187| Changed|

</details>


  <details><summary><b>Callout</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Bottom center.chromium.png|2146| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
VRs unrelated to the change in this PR